src/locales/zh-CN/request.ts

```diff
@@ -0,0 +1,19 @@
+export default {
+  'app.request.error': '请求错误',
```

Comment:
One note would be, I didn't add this file for the non `zh-CN` and `en-US` locales, I could add those with some rough translations if that would help.

Reply:
request Any development will be customised, I think it would be good to add some comments, no need to translate